accesspoc/dips/views.py

```diff
@@ -127,24 +127,32 @@ def users(request):
         'first_name': 'first_name',
         'last_name': 'last_name',
         'email': 'email',
+        'groups': 'group_names'
     }
     sort_option, sort_dir = get_sort_params(request, sort_options, 'username')
     sort_field = sort_options.get(sort_option)
     if sort_dir == 'desc':
         sort_field = '-%s' % sort_field
 
     # Search
+    class GroupsSQ(Subquery):
+        """Subquery to concatenate group names, requires MySQL or SQLite"""
+        template = "(SELECT GROUP_CONCAT(name, ', ') FROM (%(subquery)s))"
+        output_field = CharField()
+
+    subquery = GroupsSQ(Group.objects.filter(user=OuterRef('pk')))
+    users = User.objects.annotate(group_names=subquery)
     if 'query' in request.GET and request.GET['query']:
         query = request.GET['query']
-        users = User.objects.order_by(sort_field).filter(
+        users = users.order_by(sort_field).filter(
             Q(username__icontains=query) |
             Q(first_name__icontains=query) |
             Q(last_name__icontains=query) |
             Q(email__icontains=query) |
-            Q(groups__name__icontains=query)
+            Q(group_names__icontains=query)
```

Comment:
I think that you should start considering moving this queries to the model layer - it doesn't need to happen in this PR. The views should be as thin as possible. You can do all sort of things in the model layer, e.g. see [proxies](https://docs.djangoproject.com/en/dev/topics/db/models/#proxy-models).